challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
+    
+    for(let i = 0; i < integers.length - 4; i++) {
+        integers[i] = "*";
+        cardNumber += integers[i]
+    }
+   
+    for(let i = integers.length - 4; i < integers.length; i++){
+        cardNumber += integers[i]
+    }
+    console.log(cardNumber)
+}
+
+
+hideCardNumber(123838383848)
```

Comment:
FYI: There are only 12 digits in the card number. Most credit cards have 16 digits.

Good practice to test with 16 digits and validate the input as otherwise any amount of digits can be entered.

There are no validations in this function whatsoever. No defensive programming. I can pass in a string and it will still work.

E.g. hideCardNumber("Hello") will return "*ello" which is not what we want. We need to validate the input and catch the error.

Then raise an appropriate validation error message.

Please think of edge cases and validate the input. This is very important.

Reply:
@Farhanam76 This has not been addressed! I literally missed this on the cross-check because you did not address these comments formally. Luckily I checked once again.